lib/modules/side_menu/side_menu_module.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/widgets.dart';
+import 'package:survey/core/viper/module.dart';
+import 'package:survey/modules/screen.dart';
+
+class SideMenuModule extends Module {
```

Comment:
this is another example that we shouldn't include the unrelated part to the PR 🙇  maybe you should remove this out.

Reply:
So I should leave `TODO` in router?

Reply:
Anyway, please let me fix it in my next PRs 😁